cdap-ui/app/cdap/components/Footer/index.js

```diff
@@ -31,28 +31,11 @@ export default class Footer extends Component {
       <footer>
         <div className="container">
           <div className="row text-muted">
-            <div className="col-sm-6 text-uppercase">
-              <p>
-                <span className="cask-logo"></span>
-                <span>Copyright &copy; {this.copyrightYear} Cask Data, Inc.</span>
+            <div className="text-uppercase">
+              <p className="text-center">
+                <span>Copyright &copy; 2011-{this.copyrightYear} Cask Data, Inc.</span>
```

Comment:
@PGuimarais did you confirm we need the year 2011? I think all we need is the current year